pkgs/games/osu-lazer/bin.nix

```diff
@@ -64,10 +65,13 @@ else appimageTools.wrapType2 {
       contents = appimageTools.extract { inherit pname version src; };
     in
     ''
-      mv -v $out/bin/${pname} $out/bin/osu\!
-      install -m 444 -D ${contents}/osu\!.desktop -t $out/share/applications
+      . ${makeWrapper}/nix-support/setup-hook
+      mv -v $out/bin/${pname} $out/bin/osu!
+      wrapProgram $out/bin/osu! \
+        --set OSU_EXTERNAL_UPDATE_PROVIDER 1
```

Comment:
Seems to be working:
![2024-09-07-180646_hyprshot](https://github.com/user-attachments/assets/eab824f4-3ce8-47b2-b4c4-73dfd16f3c5e)